generated/nidaqmx/task.py

```diff
@@ -402,6 +402,39 @@ def is_task_done(self):
 
         return is_task_done
 
+    def perform_bridge_offset_nulling_cal(self, channel=""):
+        """
+        Performs a bridge offset nulling calibration on the channels in the task.
+        If the task measures both bridge-based sensors and non-bridge-based sensors,
+        use the channels input to specify the names of the channels that measure 
+        bridge-based sensors.
+        """
+
+        self._interpreter.perform_bridge_offset_nulling_cal(
+            self._handle, channel)
+
+    def perform_bridge_offset_nulling_cal_ex(self, channel="", skip_unsupported_channels=False):
+        """
+        Performs a bridge offset nulling calibration on the channels in the task.
+        If the task measures both bridge-based sensors and non-bridge-based sensors,
+        use the channels input to specify the names of the channels that measure 
+        bridge-based sensors.
+        """
+
+        self._interpreter.perform_bridge_offset_nulling_cal_ex(
+            self._handle, channel, skip_unsupported_channels)
```

Comment:
We should only support `perform_bridge_offset_nulling_cal_ex`, but not have the `_ex` suffix. That is an artifact of years of DAQmx history and not something we should expose in the Python API. For example, in LabVIEW, we stopped shipping the original version of the API on the palette for similar reasons. Since this is a new API, no need to have the old one at all. FYI - if its easier, keeping the `_ex` in the `_interpreter` layer is fine. I'm not sure if that is the case or not.

Reply:
Oh, I also don't mind if the interpreter ends up with unused methods, like `perform_bridge_offset_nulling_cal` that nobody uses.

Reply:
Thanks for your feedback. I have removed the perform_bridge_offset_nulling_cal and only left the `_ex ` function in the interpreter layer.